### PR TITLE
Created sample profile page

### LINKED ISSUE #2: Create user profile mockup
## Description

The first big feature we need to implement after authentication is the user profile view.
We need to figure out what (minimally) the user should have on their profile and make a mock
view of what that would look like in the app.

## Spec:

- [ ] Profile photo
- [ ] Bio
- [ ] anything else we determine; we can discuss this on the Tuesday standup

Note: there is no backend implementation yet, so for now
let's create just the UI. Once the user authentication flow
is in place, we will pull data from the backend.

### PR DESCRIPTION
**Pull request by ross3102**
## Description
Created a layout for a profile page with a user's name, profile picture, location, bio, and a list of public groups the user belongs to.

Closes #2

### Screenshot

image

### CODE REVIEW

File: screens/Profile.tsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
+import { StyleSheet, Image, ScrollView } from 'react-native';
+
+import { FontAwesome } from '@expo/vector-icons';
+import { Text, View } from '../components/Themed';
+import { RootTabScreenProps } from '../types';
+
+export default function TabOneScreen({ navigation }: RootTabScreenProps<'TabOne'>) {
```
Comment (connorlbark) [inline screens/Profile.tsx]: I feel like we should rename it away from TabOneScreen to something more descriptive.

**Comment by connorlbark** [comment]
most everything looks good! just the one comment then i'll take take a look & approve it

**Comment by nicoledanuwidjaja** [comment]
Also added the fixes for eslint to this PR as well!

File: .eslintrc.js [hunk 2]
```diff
@@ -12,6 +12,26 @@ module.exports = {
     ecmaVersion: 'latest',
     sourceType: 'module',
   },
-  plugins: ['react', '@typescript-eslint'],
-  rules: {},
+  plugins: ['react', 'react-native', '@typescript-eslint'],
+  rules: {
```
Comment (nicoledanuwidjaja) [inline .eslintrc.js:16]: These rules were determined because eslint was complaining about the way the ts starter code was set up

**Review by connorlbark — APPROVED** [review]
lgtm

**Review by jciolfi — APPROVED** [review]
lgtm !

**Pull request merged**